apps/condo/domains/common/constants/featureflags.js

```diff
@@ -30,6 +30,9 @@ const REASSIGN_EMPLOYEE_TICKETS = 'reassign-employee-tickets'
 const SNOWFLAKES_SETTINGS = 'snowflakes-settings'
 const USER_WHITE_LIST_FOR_FIND_ORGANIZATIONS_BY_TIN = 'user-white-list-for-find-organizations-by-tin'
 const METER_REPORTING_PERIOD_STRICT_RULE = 'meter-reporting-period-strict-rule'
+const SUBSCRIPTION = 'subscription'
+const CHECK_TLS_CLIENT_CERT = 'check-tls-client-cert-config'
+const ADD_SECTION_FLOOR = 'add-section-floor'
```

Comment:
Now it turns out that this feature will be disabled for those clients for whom it was enabled?
Why is this feature under the feature flag? Maybe it's time to open it up to everyone.